source/jormungandr/jormungandr/utils.py

```diff
@@ -62,6 +62,8 @@
 WEEK_DAYS_MAPPING = ("monday", "tuesday", "wednesday", "thursday", "friday", "saturday", "sunday")
 COVERAGE_ANY_BETA = "any-beta"
 BEST_BOARDING_POSITION_KEY = "{}-{}"
+ONE_DAY = 86400
```

Comment:
More pythonic: You can probably use `datetime.timedelta(days=1).total_seconds()` instead of `86400`.